doc/source/dev/reviewer_guidelines.rst

```diff
@@ -0,0 +1,135 @@
+
+Who can be a reviewer?
+======================
+
+You don't have to be a project member to contribute reviews. Consider being a
+reviewer if 
+
+- You have contributed to the NumPy project before, and feel like you can
+  comment on technical or organizational aspects of the PR; 
+- You haven't contributed before, but are an experienced NumPy user, and see a
+  pull request that touches a funcionality that you use often;
+- You can contribute comments about content, style, organization or good
+  documentation practices.  
+
+If we do not know you yet, consider introducing yourself before you start
+reviewing pull requests.
+
+Communication Guidelines
+========================
+
+- Every PR, good or bad, is an act of generosity. Opening with a positive
+  comment will help the author feel rewarded, and your subsequent remarks may be
+  heard more clearly. You may feel good also.
+- Begin if possible with the large issues, so the author knows they've been
+  understood. Resist the temptation to immediately go line by line, or to open
+  with small pervasive issues.
+- You are the face of the project, and NumPy some time ago decided `the kind of
+  project it will be <https://numpy.org/code-of-conduct/>`_: open, empathetic,
+  welcoming, friendly and patient. Be `kind
+  <https://youtu.be/tzFWz5fiVKU?t=49m30s>`_ to contributors.
+- Do not let perfect be the enemy of the good, particularly for documentation.
+  If you find yourself making many small suggestions, either open a PR against
+  the original branch, push changes to the contributor branch, or merge the PR
+  and then open a new PR against upstream.
+- If you need help writing replies in reviews, check out some `Standard replies
+  for reviewing
+  <https://scikit-learn.org/stable/developers/tips.html#saved-replies>`_.
+
+Reviewer Checklist
+==================
+
+- Is the intended behavior clear under all conditions (e.g. unexpected inputs
+  like empty arrays or nan/inf values)?
+- Is the code easy to read and low on redundancy? Should variable names be
+  improved for clarity or consistency? Should comments be added? Should comments
+  be removed as unhelpful or extraneous?
+- Does the documentation follow the :ref:`NumPy guidelines<howto-document>`? Are
+  the docstrings properly formatted?
+- If it is not obvious from the PR description, add a short explanation of what
+  a branch did to the merge message and, if closing an issue, also add "Closes
+  #123" where 123 is the issue number.
+- For code changes, at least one core team member (those with commit rights)
+  should review and approve all pull requests. If you are the first to review a
+  PR and approve of the changes use the GitHub `approve review
+  <https://help.github.com/articles/reviewing-changes-in-pull-requests/>`_ tool
+  to mark it as such. If you are a subsequent reviewer, please approve the
+  review or, if you have commit rights and think no more review is needed, merge
+  the PR.
+
+Core team members
```

Comment:
Can we change this to "For maintainers"? I really dislike the term "core team" (implies that some type of activities are more important than others) and want to get rid of it soon.